doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -271,6 +271,27 @@ Cellular samples
 
     * The Thingy:91 X build to support Wi-Fi by default without overlays.
 
+ * Updated the :kconfig:option:`CONFIG_NRF_CLOUD_CHECK_CREDENTIALS` Kconfig option to be optional and can be disabled for the following samples:
+
+   * :ref:`nrf_cloud_rest_cell_pos_sample`
+   * :ref:`nrf_cloud_rest_device_message`
+   * :ref:`nrf_cloud_rest_fota`
+
```

Comment:
Could move the these whole section after line no: 267 (after "Cellular samples" section header), since these entries impact more that one cellular samples.